src/Provider/ImageProvider.php

```diff
@@ -179,6 +179,11 @@ public function generatePublicUrl(MediaInterface $media, $format)
             $path = $this->thumbnail->generatePublicUrl($this, $media, $format);
         }
 
+        // if path is url, do not action
```

Comment:
It's usually "do not" + verb, and "action" is a noun

Reply:
Not my words ☺, but I will review this comment.